substrate/frame/examples/basic/src/lib.rs

```diff
@@ -38,18 +38,32 @@
 //!
 //! This pallet provides basic examples of using:
 //!
+//! - Pallet hooks to implement some logic that's executed before and after a block is imported
+//!   (see: [`frame_support::traits::Hooks`])
+//! - The [`StorageValue`](frame_support::storage::types::StorageValue) API to demonstrate it's use
+//!   of `mutate` on:
+//! 	- A storage value that stores some `Balance` and uses the default
+//!    [`OptionQuery`](frame_support::storage::types::OptionQuery) which will always return the
+//!    value in storage when queried
```

Comment:
What will this return if the value doesn't exist? `None`. Let's clarify that. OptionQuery return `Option<T>` and `None`, `ValueQuery` returns `T` and `Default::default`.